packages/compass-preferences-model/src/preferences.ts

```diff
@@ -119,10 +104,11 @@ type PostProcessFunction<T> = (
 ) => T;
 
 type PreferenceDefinition<K extends keyof AllPreferences> = {
-  /** The type of the preference value, in Ampersand naming */
-  type: AmpersandType<AllPreferences[K]>;
+  /** The type of the preference value */
+  type: PreferenceValue<AllPreferences[K]>;
   /** An optional default value for the preference */
   default?: AllPreferences[K];
+  // todo: do we need it
   /** Whether the preference is required in the Ampersand model */
   required: boolean;
```

Comment:
Its safe to remove this property now?

Reply:
It's similar to `type`, `default`, and `values`, right? They were all used by Ampersand for validation, that's a featur that's going away here